talk/owt/sdk/conference/conferencewebtransportchannel.cc

```diff
@@ -100,9 +100,9 @@ void ConferenceWebTransportChannel::Connect() {
   int fingerprint_idx = 0;
   for (auto fingerprint : trusted_fingerprints) {
     quic_params.server_certificate_fingerprints[fingerprint_idx]->fingerprint =
-        new char[fingerprint.length()];
+        new char[fingerprint.length()+1];
     memcpy((void*)(quic_params.server_certificate_fingerprints[fingerprint_idx]->fingerprint),
```

Comment:
memset after allocation.

Reply:
I believe in modern C++ this will be zero-initialized by default, as a value-type array: https://en.cppreference.com/w/cpp/language/zero_initialization